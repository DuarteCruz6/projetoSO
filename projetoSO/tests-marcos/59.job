WRITE [(e6,93)(h9,71)]
DELETE [h9]
READ [e6]
READ [e6]
WRITE [(r5,9)]
WRITE [(k1,18)(e5,84)(w8,24)]
WRITE [(d5,4)(z0,7)(l5,31)(h4,13)(f3,11)]
DELETE [h4,w8,z0,k1,r5]
READ [e5]
WRITE [(m3,25)(i1,94)]
SHOW
DELETE [e6]
WRITE [(h9,10)(j3,55)(t4,98)(c5,58)(a1,38)]
WRITE [(z1,4)]
WRITE [(g6,42)(n9,53)(m5,47)(a3,89)(z7,64)]
BACKUP
WRITE [(q8,94)]
BACKUP
WRITE [(q7,50)(t0,80)(l3,28)(h3,5)]
READ [q7]
WRITE [(v0,9)(p9,76)(x7,53)(i7,65)]
WRITE [(f1,33)(h0,59)(i8,4)(x2,38)(o2,35)]
WRITE [(y1,22)(v8,5)(a0,16)(d8,1)]
WRITE [(l9,66)]
WRITE [(d7,82)]
WRITE [(u2,54)]
SHOW
READ [z1]
WRITE [(t9,27)(w0,69)(i4,75)(l1,74)]
WRITE [(g3,37)(x0,0)(b7,9)(f9,28)(u7,54)]
WRITE [(r4,60)(y5,100)(d4,7)(m4,94)]
WRITE [(h7,77)(x9,62)(s3,1)(w9,15)(k8,71)]
DELETE [f1,t9,c5,j3]
WRITE [(b6,21)]
READ [y5]
SHOW
WRITE [(s6,52)]
WRITE [(y2,44)(b3,61)(v6,60)(j8,6)]
SHOW
READ [l1]
WRITE [(a8,87)(x5,28)(z0,93)(t5,74)]
READ [h0]
WRITE [(y0,53)(n6,54)(o0,29)]
WRITE [(u3,47)(u8,63)]
WRITE [(o9,85)(a5,86)(k9,95)]
WRITE [(i9,16)(c6,70)]
DELETE [y1]
