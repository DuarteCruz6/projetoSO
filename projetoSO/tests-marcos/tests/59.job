WRITE [(r7,35)(u8,88)(w3,49)(u1,8)]
WRITE [(x4,86)(y0,51)(z6,62)(p6,63)]
WRITE [(u2,86)]
READ [x4]
DELETE [u1,u8,w3,p6,x4]
WRITE [(a6,79)]
WRITE [(y1,63)(f2,32)]
DELETE [f2,r7,y1,y0]
READ [z6]
SHOW
READ [z6]
BACKUP
WRITE [(h0,42)(n2,37)]
DELETE [n2,h0]
WRITE [(z3,78)]
WRITE [(j9,96)(c7,10)]
READ [z3]
SHOW
READ [u2]
WRITE [(n0,63)(s7,10)]
WRITE [(f1,69)(d3,52)]
READ [z6]
WRITE [(y9,55)(m6,99)(q7,61)]
READ [a6]
BACKUP
DELETE [a6,u2,z3,d3,y9]
WRITE [(u1,48)(t4,40)]
READ [c7]
WAIT 2000
WAIT 3000
READ [f1]
READ [m6]
