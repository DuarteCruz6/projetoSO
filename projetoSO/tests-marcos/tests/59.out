[(x4,86)]
[(z6,62)]
(a6, 79)
(u2, 86)
(z6, 62)
[(z6,62)]
[(z3,78)]
(a6, 79)
(c7, 10)
(j9, 96)
(u2, 86)
(z3, 78)
(z6, 62)
[(u2,86)]
[(z6,62)]
[(a6,79)]
[(c7,10)]
[(f1,69)]
[(m6,99)]
